### PR TITLE
Small changes to README

### PR DESCRIPTION
**Pull request by ashuio**
Signed-off-by: Ashutosh Verma <mail@ashu.io>

Updates README with small changes

* Resolves: # 
* Target version: master

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,18 +8,20 @@
 
 ![](img/screenshots/screenshot1.png)
 
-- **📝 Focused writing:** No distractions, only the formatting you need.
+## Features
+
+- **📝Simple Focused writing:** No distractions, only the formatting you need.
```
Comment (juliushaertl) [inline README.md:13]:
```suggestion
- **📝 Simple focused writing:** No distractions, only the formatting you need.
```

Comment (ashuio) [reply README.md]: Done, Thanks for pointing out

Comment (juliushaertl) [reply README.md]: Sorry still missing the space compared to the other list entries between the emoji and the text :wink: Otherwise :+1:

Comment (ashuio) [reply README.md]: Thanks , That's embarrassed lol

**Review by jancborchardt — APPROVED** [review]
Looks good, thanks for the fix @ashuio! :)

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**